src/datepicker/styled-components.js

```diff
@@ -15,16 +15,15 @@ export const StyledRoot = styled<SharedStylePropsT>('div', props => {
   const {
     $theme: {typography, colors, borders},
   } = props;
-  const borderRadius = borders.useRoundedCorners ? borders.radius200 : 0;
   return {
     ...typography.font400,
     color: props.$theme.colors.datepickerDayFont,
     backgroundColor: colors.datepickerBackground,
     textAlign: 'center',
-    borderTopLeftRadius: borderRadius,
-    borderTopRightRadius: borderRadius,
-    borderBottomRightRadius: borderRadius,
-    borderBottomLeftRadius: borderRadius,
+    borderTopLeftRadius: borders.cardBorderRadius,
+    borderTopRightRadius: borders.cardBorderRadius,
+    borderBottomRightRadius: borders.cardBorderRadius,
+    borderBottomLeftRadius: borders.cardBorderRadius,
```

Comment:
Looking at this change I wonder what's the value of keeping the `useRoundedCorners` if we removed the usage of it?

Reply:
Ah, it does still get used in some places. The checkbox, datepicker range, progress bar, slider, and tag. I can't remove it outright without creating a breaking style change (either for people who have it true or those who have it false).

Reply:
This PR does not fix the confusion around `useRoundedCorners` anymore. I think we will need to wait until a future major release to remove it outright.